addons/survey/models/survey_question.py

```diff
@@ -184,6 +186,18 @@ def _check_question_type_for_pages(self):
     # COMPUTE METHODS
     # -------------------------------------------------------------------------
 
+    @api.depends('suggested_answer_ids', 'suggested_answer_ids.value')
+    def _compute_has_image_only_suggested_answer(self):
+        empty_answer_domain = [('question_id', 'in', self.ids), ('value', 'in', [False, ''])]
+        question_n_empty_answer = {
+            question.id: count
+            for question, count in self.env['survey.question.answer']._read_group(
+                empty_answer_domain, groupby=['question_id'], aggregates=['__count'])
+        }
+        for question in self:
+            question.has_image_only_suggested_answer = (
```

Comment:
Is it true ? I seems to compute "have at least one suggested answer with image and not value", but you may have answers with both at the same time on the question, isn't it ?